cluster/saltbase/salt/etcd/etcd.manifest

```diff
@@ -57,7 +59,7 @@
   },
   { "name": "varlogetcd",
     "hostPath": {
-        "path": "/var/log/etcd.log"}
+        "path": "/var/log/etcd{{ suffix }}.log"}
```

Comment:
OK - never mind. I see that you're separating the directory below (you have /var/etcd/data and /var/etcd/data{{ suffix }}.